Gildas dec17a build failure

Gildas dec17a fails to build with the following error message:

```
gcc -c -I../  -Wall    -I/tmp/nix-build-gildas-20171201_a.drv-6/gildas-src-dec17a/integ/x86_64-debian8-gfortran-openmp/include -pipe -fPIC -Dlinux -DLINUX -DPNG -O -Dunderscore -DGAG_USE_GTK -DGAG_USE_OPENMP -DGAG_USE_POSIX -DGAG_USE_PYTHON -DGAG_USE_UNDERSCORE sic_python.c -o built/x86_64-debian8-gfortran-openmp/sic_python.o
sic_python.c:11:20: fatal error: Python.h: No such file or directory
 #include <Python.h>
                    ^
compilation terminated.
```

The header file is found by the configure script, but the include flag isn't passed along.

--- a/ciment/gildas/default.nix
+++ b/ciment/gildas/default.nix
@@ -21,7 +21,6 @@ stdenv.mkDerivation rec {
   configurePhase=''
     substituteInPlace admin/wrapper.sh --replace '%%OUT%%' $out
     source admin/gildas-env.sh -b gcc -c gfortran -o openmp
-    export GAG_INC_FLAGS=""
     echo "gag_doc:        $out/share/doc/" >> kernel/etc/gag.dico.lcl
   '';
 

